src/launcher/containers/WithScrollbarContainer.jsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+import { bool, node } from 'prop-types';
+
+const WithScrollbarContainer = ({ children, hasFilter = false }) => {
+    if (hasFilter)
+        return (
+            <div className="with-scrollbar filter-adjusted-height">
+                <div className="content-container">{children}</div>
+            </div>
+        );
+    return (
+        <div className="with-scrollbar">
```

Comment:
You can just conditionally add the `filter-adjusted-height` here:
```suggestion
    return (
        <div className={`with-scrollbar ${hasFilter ? ' filter-adjusted-height' : ''}`}>
```

Or use the [`classNames`](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/e6c7739978955c8c9fa7a4a6d20f1d91b7a5702f/src/utils/classNames.ts) utility from `shared`.